src/crypto/index.js

```diff
@@ -246,6 +246,18 @@ Crypto.prototype.uploadDeviceKeys = function() {
     });
 };
 
+/**
+ * Stores the current key count which will be handled postponed (in a call of
```

Comment:
'postponed' is a verbal adjective, you can't use it as an adverb to modify 'handled' :).  s/postponed/later/.

It would be worth mentioning here where the key count is coming from: the /sync response.